CppCoreGuidelines.md

```diff
@@ -5616,7 +5616,7 @@ An initialization explicitly states that initialization, rather than assignment,
         // ...
     };
 
-    class C {   // UGLY, aka very bad
```

Comment:
These three comments were a joke on the movie titled "The Good, the Bad, and the Ugly"... we don't think this is likely to be viewed as offensive, and prefer the original.

Reply:
reverted